lldb/include/lldb/Target/Target.h

```diff
@@ -579,6 +587,11 @@ class EvaluateExpressionOptions {
   mutable std::string m_pound_line_file;
   mutable uint32_t m_pound_line_line = 0;
   bool m_prepare_playground_stub_functions = true;
+
+  ///< During expression evaluation, any SymbolContext in this list will be
+  ///< used for symbol/function lookup before any other context (except for
+  ///< the module corresponding to the current frame).
```

Comment:
Can you fix this comment upstream? It's attached to the previous declaration.

Reply:
https://github.com/llvm/llvm-project/pull/132910